api/src/main/java/org/openmrs/Relationship.java

```diff
@@ -9,27 +9,53 @@
  */
 package org.openmrs;
 
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
 import java.util.Date;
 
 /**
  * Relationship
  */
+@Entity
+@Table(name = "relationship")
 public class Relationship extends BaseChangeableOpenmrsData {
 	
 	public static final long serialVersionUID = 323423L;
 	
 	// Fields
-	
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "relationship_id_seq")
```

Comment:
Could there be any reason why you did not use `GenerationType.IDENTITY`?

Reply:
@dkayiwa  The earlier implementation utilized GenerationType.SEQUENCE, and this choice is also reflected in [this pull request](https://github.com/openmrs/openmrs-core/pull/4388/files#72), which I used as a reference. Some of the code reviews suggested a modification from GenerationType.IDENTITY to GenerationType.SEQUENCE. 

 Please let me know, if I need to change `sequence` to `Identity`.